lib/kubernetes-deploy/restart_task.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+require 'kubernetes-deploy/kubeclient_builder'
+
+module KubernetesDeploy
+  class RestartTask
+    include KubernetesDeploy::KubeclientBuilder
+
+    HTTP_OK_RANGE = 200..299
+
+    def initialize(context:, namespace:, logger:)
+      @context = context
+      @namespace = namespace
+      @logger = logger
+      @v1beta1_kubeclient = build_v1beta1_kubeclient(context)
+    end
+
+    def perform(deployments)
+      if deployments.empty?
+        raise ArgumentError, "#perform takes at least one deployment to restart"
+      end
+
+      fetch_deployments(deployments).each do |record|
```

Comment:
You're right, I should have set the message to `triggering restart of "web"` because it doesn't way for the restart to finish. The pods are recreated according to the rollout strategy that every project has defined.

Do you think the tool needs the `--wait` flag to wait for the restart? I'm not sure about that because that's something that can be always watched in kubernetes dashboard.